R/split.R

```diff
@@ -29,7 +35,16 @@
 #'
 #' adlb <- xportr_split(adlb, "LBCAT")
 xportr_split <- function(.df, split_by = NULL) {
-  attr(.df, "_xportr.split_by_") <- split_by
+  lifecycle::deprecate_warn(
+    when = "0.5.0",
+    what = "xportr_split()",
+    with = "xportr_write()",
+    details = "Please use the argument `max_gb_size` in the
+    function xportr_write() instead` instead."
+  )
```

Comment:
love this!  ty for doing this!

Reply:
Usually, the deprecated functionality is still available in Phase 1 (deprecation warning). However, here `xportr_split()` still works as before but in `xportr_write()` it is ignored. I.e., the functionality is already removed.

I would either go directly to Phase 2 (deprecation error) or keep the old functionality (in addition to the new functionality) in `xport_write()`. I assume `xportr_split()` is used only by a few users (if at all). Thus I would tend to the first option.

@bms63 , what do you think?